Tests/Editor/ValidatorBlacklistedClassTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace DTValidator.TestBlacklisted
```

Comment:
This namespace shouldn't be necessary since `ValidatorBlacklistedClass` should operate on the _class_ not the namespace, I think you can remove.

Reply:
That made it clear. Thanks a lot. Apparently I'm not as proficient in namespaces as I thought.